src/utils/hwManager.js

```diff
@@ -1,12 +1,16 @@
 // istanbul ignore file
 // TODO include unit test
+import to from 'await-to-js';
+import i18next from 'i18next';
 import {
   getPublicKey,
   signTransaction,
   subscribeToDeviceConnceted,
   subscribeToDeviceDisonnceted,
   subscribeToDevicesList,
 } from '../../libs/hwManager/communication';
+import { calculateTxId, createSendTX, createRawVoteTX } from './rawTransactionWrapper';
```

Comment:
I think `./rawTransactionWrapper';` can be deleted and replaced by use of `transactions.utils.getTransactionId(...)`, `transactions.transfer(...)`, and `transactions.castVotes(...)`  from `list-elements`.

The only problem I saw is that you cannot set `senderPublicKey` other than by `passphrase`, but it can be easily added separately, like `id` and `signature`.

https://lisk.io/documentation/lisk-sdk/lisk-elements/packages/transactions#methods-for-creating-signature-objects

Also mentioned in 
https://github.com/LiskHQ/lisk-hub/blob/f75c0663f2fc6b5b27023fe00b6a8f7f650e9d94/src/utils/rawTransactionWrapper.js#L4-L5